lib/Dialect/FIRRTL/Transforms/BlackBoxMemory.cpp

```diff
@@ -326,7 +326,7 @@ replaceMemWithWrapperModule(DenseMap<MemOp, FModuleOp, MemOpInfo> &knownMems,
     memOp->erase();
 }
 
-static void
+static bool
```

Comment:
Can you add a comment documenting the return type here? (It's documented in the body, but it would be good to be able to just read it here.)